charts/kubescape-cloud-operator/values.yaml

```diff
@@ -424,6 +424,8 @@ otelCollector:
     host: ""
     port: 4317
     insecure: true
+    headers:
```

Comment:
these are only required when submitting to uptrace, I think it's better to not include them in `values.yaml`

Reply:
Noted. I [removed](https://github.com/kubescape/helm-charts/pull/64/commits/a7fffb2bc147473c76de24f3015595b4cbdeafe0) the headers from the example